src/routes/contact.js

```diff
@@ -57,14 +57,14 @@ router.delete('/', auth, async (req, res) => {
 router.get('/', auth, async (req, res) => {
 	try {
 		const user = await User.findOne({ uid: req.user.uid }).select('-password');
-		if (user.role != 'admin') res.status(401).json({ msg: 'Not authorized' });
+		if (user.role != 'admin') res.status(401).json({ code: 'QUERY_FETCH_UNAUTHORISED', msg: 'Not authorized' });
 		else {
 			const dContact = await Contact.find({});
 			if (dContact) res.json(dContact);
-			else res.json({ msg: `Error in fetching` });
+			else res.status(417).json({ code: 'QUERY_FETCH_ERROR', msg: `Error in fetching` });
```

Comment:
Why `417`?

Reply:
Missed that it does not take a headers, fixed.